shared/localization/locales/es-419.json

```diff
@@ -1038,7 +1038,7 @@
     "message": "La URL del manifiesto cambió mientras este se estaba recuperando."
   },
   "core/audits/installable-manifest.js | manifest-missing-name-or-short-name": {
-    "message": "El manifiesto no contiene los campos \"name\" ni \"short_name\""
```

Comment:
ah we [JUST changed this](https://github.com/GoogleChrome/lighthouse/commit/fa7031796e5b63a2b0fe6e676685bfdffa780a04) so we probably are doing an import a tad too quickly?

Reply:
I'm fine to defer the release for another day to ensure we get the translated versions. In the future, maybe we should build in a 1-2 day buffer between string export and string import before release?